v2/futures/exchange_info_service.go

```diff
@@ -67,12 +67,14 @@ type Symbol struct {
 	UnderlyingSubType     []string                 `json:"underlyingSubType"`
 	SettlePlan            int64                    `json:"settlePlan"`
 	TriggerProtect        string                   `json:"triggerProtect"`
-	OrderType             []OrderType              `json:"OrderType"`
+	OrderType             []OrderType              `json:"orderType"`
```

Comment:
could you add unit test for these changes?

Reply:
Yep. Done